common/scripted_score_values/wc_scripted_generic_artifact_spawn_values.txt

```diff
@@ -0,0 +1,774 @@
+# Drop chances
+
+generic_artifact_rarity_legendary_score = {
+	# modifier = { # In-game chances to spawn
+		# factor = 1
+
+		# check_variable = { which = local_new_generic_artifact_gamestart_variable value < 1 }
+		# check_variable = { which = local_new_generic_artifact_gamestart_variable value < 1 }
+	# }
+
+	modifier = {
+		factor = 0
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value = 1 }
+		has_game_rule = { name = artifact_spawn_rarity value = even_less_random }
+		lower_real_tier_than = king
+	}
+
+	modifier = {
+		factor = 5
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value = 1 }
+		NOT = { has_game_rule = { name = artifact_spawn_rarity value = random } }
+		tier = emperor
+	}
+	modifier = {
+		factor = 4
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value = 1 }
+		NOT = { has_game_rule = { name = artifact_spawn_rarity value = random } }
+		tier = king
+	}
+	modifier = {
+		factor = 3
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value = 1 }
+		NOT = { has_game_rule = { name = artifact_spawn_rarity value = random } }
+		tier = duke
+	}
+	modifier = {
+		factor = 2
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value = 1 }
+		NOT = { has_game_rule = { name = artifact_spawn_rarity value = random } }
+		tier = count
+	}
+}
+generic_artifact_rarity_epic_score = {
+	modifier = { # In-game chances to spawn
+		factor = 2
+
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value < 1 }
+		check_variable = { which = local_new_generic_artifact_gamestart_variable value < 1 }
```

Comment:
The same variable checked twice? What it's a point?

Reply:
Probably forgot to remove it after changing variables
Will do it later

Reply:
Done